app/questionnaire/questionnaire_schema.py

```diff
@@ -577,22 +598,24 @@ def _update_dependencies_for_value_source(
             self._answer_dependencies_map[value_source["identifier"]] |= {
                 self._get_dependent_for_block_id(block_id=block_id, answer_id=answer_id)
             }
-        if value_source["source"] == "calculated_summary":
+        elif value_source["source"] == "calculated_summary":
             identifier = value_source["identifier"]
-            if calculated_summary_block := self.get_block(identifier):
-                answer_ids_for_block = get_calculated_summary_answer_ids(
-                    calculated_summary_block
-                )
-                dependent = self._get_dependent_for_block_id(
-                    block_id=block_id, answer_id=answer_id
+            self._update_dependencies_for_calculated_summary_value_source(
+                calculated_summary_id=identifier, block_id=block_id, answer_id=answer_id
+            )
+
+        elif value_source["source"] == "grand_calculated_summary":
+            identifier = value_source["identifier"]
+            # Type ignore: validator will ensure identifier is valid
+            gcs_block: ImmutableDict = self.get_block(identifier)  # type: ignore
```

Comment:
As per the other PR, there is mixture here of shortened and the full version of `gcs` in vars which could probably do with being made consistent throughout.

Reply:
Updated them all to full names